scripts/kube-deepcopy.sh

```diff
@@ -8,4 +8,4 @@ set -eu
 # files. We want to exclude all kube-related code from plan9 builds because some
 # apimachinery libraries refer to syscalls that are not available for plan9
 # https://github.com/kubernetes/apimachinery/blob/v0.28.2/pkg/util/net/util.go#L42-L63 
-sed -i "1 s|$| \&\& \!plan9|" k8s-operator/apis/v1alpha1/zz_generated.deepcopy.go
+sed -i.bak "1 s|$| \\&\\& \\!plan9|" k8s-operator/apis/v1alpha1/zz_generated.deepcopy.go && rm k8s-operator/apis/v1alpha1/zz_generated.deepcopy.go.bak
```

Comment:
Why this change? Did the command error on your machine?

Reply:
`sed -i` requires an extension on FreeBSD/OS X mac, this makes it work for both Linux and OS X. If I run this on mac right now with the default `sed` I get:

```
sed: 1: "k8s-operator/apis/v1alp ...": invalid command code k
```

I can switch to GNU Sed on my machine, but figure this helps eveyrone

Reply:
Makes sense 👍🏼 I assumed it was a Mac thing, but wanted to clarify